### PR TITLE
Doc cine update jan21

### PR DESCRIPTION
**Pull request by bryanzakounity**
added Confiner2d .md to toc
updated existing images to CM new look and feel
updated text in existing pages to reflect any changes from new CM look and feel
added missing field/properties to existing pages

### CODE REVIEW

File: Documentation~/CinemachineConfiner2D.md [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
+In these cases, for efficiency reasons, Cinemachine does not automatically regenerate the inner polygon. The client needs to call the InvalidateCache() method to trigger the recalculation. You can do this from; 
+
+- the script by calling InvalidateCache, or 
+- the component inspector; to do so, press the **Invalidate Cache** button.
 
 If the input polygon scales uniformly or translates or rotates, the cache remains valid. 
 
 ## Oversize Windows
-In the event that there are sections of the confining polygon that are too small to fully contain the camera window, then a _polygon skeleton_ will be calculated for those regions.  This is a zero-area shape that serves as a place to put the camera when it is confined to this region of the shape.  
+Oversize WindowIf sections of the confining polygon are too small to fully contain the camera window, Cinemachine calculates a polygon skeleton for those regions. This is a shape with no area, that serves as a place to put the camera when it is confined to this region of the shape.
```
Comment (gaborkb) [inline Documentation~/CinemachineConfiner2D.md:22]: Space missing

File: Documentation~/CinemachineVirtualCamera.md [hunk 2]
```diff
@@ -34,7 +35,10 @@ At any time, each Virtual Camera may be in one of these states:
 | | _Orthographic Size_ | When using an orthographic camera, defines the half-height of the camera view, in world coordinates. Available when the Unity camera with the Cinemachine Brain component uses a Projection of Orthographic. |
 | | _Near Clip Plane_ | The closest point relative to the camera where drawing occurs. |
 | | _Far Clip Plane_ | The furthest point relative to the camera where drawing occurs. |
+| | _Mode Override_ | Controls whether the Perspective/Orth, SensorSize, and GateFit are set in the Camera object, or are overidden in the virtual camera component. |
 | __Dutch__ || Dutch angle. Tilts the Unity camera on the z-axis, in degrees. This property is unique to the Virtual Camera; there is no counterpart property in the Unity camera. |
+| __Blend Hint__ || Provides hints for blending positions to and from the virtual camera. |
+| __Inherit Position__ || When enabled, whenever this virtual camera goes live, forces the initial position to be the same as the current position of the Unity Camera. |
```
Comment (gaborkb) [inline Documentation~/CinemachineVirtualCamera.md:41]: When enabled, whenever this virtual camera goes live, forces the initial position to be the same as the current position of the Unity Camera**, if possible.**

File: Documentation~/CinemachineVirtualCameraBody.md [hunk 3]
```diff
@@ -6,9 +6,23 @@ Use the Body properties to specify the algorithm that moves the Virtual Camera i
...
-* [__Transposer__](CinemachineBodyTransposer.html): moves in a fixed relationship to the __Follow__ target.
-* [__Do Nothing__](CinemachineBodyDoNothing.html): does not move the Virtual Camera.
-* [__Framing Transposer__](CinemachineBodyFramingTransposer.html): moves in a fixed screen-space relationship to the __Follow__ target.
-* [__Orbital Transposer__](CinemachineBodyOrbitalTransposer.html): moves in a variable relationship to the __Follow__ target, optionally accepting player input.
-* [__Tracked Dolly__](CinemachineBodyTrackedDolly.html): moves along a predefined path.
-* [__Hard Lock to Target__](CinemachineBodyHardLockTarget.html): uses the same position at the __Follow__ target.
+
+
+- [__Do Nothing__](CinemachineBodyDoNothing.html): does not move the Virtual Camera.
+- 3rd Person follow: Pivots the camera horizontally and vertically around the player, to the __Follow__ target.
```
Comment (gaborkb) [inline Documentation~/CinemachineVirtualCameraBody.md:12]: This seems to be missing some formating

File: Documentation~/images/CinemachineConfiner2Dc.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+In these cases, for efficiency reasons, Cinemachine does not automatically regenerate the inner polygon. The client needs to call the InvalidateCache() method to trigger the recalculation. You can do this from; 
+
+- the script by calling InvalidateCache, or 
+- the component inspector; to do so, press the **Invalidate Cache** button.
+
+If the input polygon scales uniformly or translates or rotates, the cache remains valid. 
+
+## Oversize Windows
+Oversize WindowIf sections of the confining polygon are too small to fully contain the camera window, Cinemachine calculates a polygon skeleton for those regions. This is a shape with no area, that serves as a place to put the camera when it is confined to this region of the shape.
```
Comment (gaborkb) [inline Documentation~/images/CinemachineConfiner2Dc.md:22]: Space missing before If

**Review by gaborkb — CHANGES_REQUESTED** [review]
Minor changes needed.

File: Documentation~/images/CinemachineConfiner2Dc.md (hunk 4 above)
Comment (bryanzakounity) [reply Documentation~/images/CinemachineConfiner2Dc.md:22]: Hmmm... this file isn't to be used. Where did you find it?

File: Documentation~/CinemachineConfiner2D.md (hunk 1 above)
Comment (bryanzakounity) [reply Documentation~/CinemachineConfiner2D.md:22]: Done.

Comment (bryanzakounity) [reply Documentation~/CinemachineConfiner2D.md:22]: Done.

File: Documentation~/CinemachineVirtualCameraBody.md (hunk 3 above)
Comment (bryanzakounity) [reply Documentation~/CinemachineVirtualCameraBody.md:12]: Done.

File: Documentation~/CinemachineVirtualCamera.md (hunk 2 above)
Comment (bryanzakounity) [reply Documentation~/CinemachineVirtualCamera.md:41]: Done.

**Review by gaborkb — APPROVED** [review]

**Pull request merged**